### PR TITLE
ADEN-6077 Kill BTF slots from creative even for PAL pages

### PR DESCRIPTION
**Pull request by v3rth**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/provider/btfBlocker.js [hunk 1]
```diff
@@ -71,16 +75,15 @@ define('ext.wikia.adEngine.provider.btfBlocker', [
...
+			var context = adContext.getContext();
 
 			log('startBtfQueue', log.levels.info.debug, logGroup);
 
 			if (btfQueueStarted) {
 				return;
 			}
 
-			if (context.opts.premiumAdLayoutEnabled && !roadblockBlocksBtf) {
+			if (context.opts.premiumAdLayoutEnabled && !isBTFDisabledByCreative()) {
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/provider/btfBlocker.js:86]: Can we, therefore, remove: https://github.com/Wikia/app/blob/9621f1a95c03f77d3910f6cd054351d0384220c5/extensions/wikia/AdEngine/js/context/uapContext.js#L39?

Comment (nandy-andy) [reply extensions/wikia/AdEngine/js/provider/btfBlocker.js:86]: And was this the actual bug? Because everything else looks cosmetic + unit tests :+1: :)

**Review by nandy-andy — APPROVED** [review]

File: extensions/wikia/AdEngine/js/provider/btfBlocker.js (hunk 1 above)
Comment (kvas-damian) [reply extensions/wikia/AdEngine/js/provider/btfBlocker.js:86]: I believe this change breaks kill code in roadblock. By design it should block all slots except `TOP_RIGHT_BOXAD`. In this case all premium slots will be unblocked.

Comment (kvas-damian) [reply extensions/wikia/AdEngine/js/provider/btfBlocker.js:86]: Ups, I was wrong :)

**Pull request merged**